### PR TITLE
New package: cobalt-org.cobalt version 0.19.6

### PR DESCRIPTION
**Pull request by joshlangley**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/173291)

### CODE REVIEW

**Comment by joshlangley** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\cobalt-org.cobalt__DefaultSource\cobalt.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\cobalt.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

File: manifests/c/cobalt-org/cobalt/0.19.6/cobalt-org.cobalt.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- RelativeFilePath: cobalt.exe
+  PortableCommandAlias: cobalt
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+Commands:
+- cobalt
+ReleaseDate: 2024-08-07
+Installers:
```
Comment (stephengillie) [inline manifests/c/cobalt-org/cobalt/0.19.6/cobalt-org.cobalt.installer.yaml:18]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Automatic Validation ended with:
> 
> > Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\cobalt-org.cobalt__DefaultSource\cobalt.exe returned exit code: -1073741515
> > Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\cobalt.exe returned exit code: -1073741515
> 
> Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).
> 
> (Automated response - build 895.)


Manual Validation ended with:
image

File: manifests/c/cobalt-org/cobalt/0.19.6/cobalt-org.cobalt.installer.yaml (hunk 1 above)
Comment (joshlangley) [reply manifests/c/cobalt-org/cobalt/0.19.6/cobalt-org.cobalt.installer.yaml:18]: Hello @stephengillie ! I'm new to WinGet packaging and am unsure how to replicate this? On my system the package installs and works just fine with `winget install --manifest <snip>/manifests/c/cobalt-org/cobalt/0.19.6/`. What am I doing wrong?

**Comment by joshlangley** [comment]
I would also like to clarify, is the "Publisher" field supposed to be the person who published the package to WinGet or the person who released it to the world?

**Comment by stephengillie** [comment]
I think the `Publisher` field has to match the ARP/ANF package display name in Registry. If it doesn't, then I think a `DisplayName` is needed.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image

**Pull request merged**